frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -660,7 +681,139 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = (props) => {
     if (!_.isString(group) || !_.isString(groupType)) {
       return null;
     }
-    return _.startCase(group.split(groupType)[1]);
+    let groupName = group.split(groupType)[1];
+    if (groupType.startsWith(SpecCapability.arrayFieldGroup)) {
+      [groupName] = groupName.split(':');
+    }
+    return groupName;
+  };
+
+  const addFieldsToGroup = (fieldList, group) => {
+    const newFormValues = _.cloneDeep(formValues);
+    const groupName = getGroupName(group, SpecCapability.arrayFieldGroup);
+    const newFields = _.cloneDeep(fields);
+
+    if (_.isArray(fieldList) && !_.isEmpty(fieldList) && !_.isNil(group)) {
+      _.forEach(fieldList, (field) => {
+        const newField = _.cloneDeep(field);
+        const pathInfoList = newField.path.split('.');
+        const groupInfo = pathInfoList.find((ele) => ele.startsWith(groupName));
+        const groupNumIndex = pathInfoList.indexOf(groupInfo);
+        const groupIndex = group.split(groupName.concat(':'))[1];
+        const newGroupName = groupName
+          .concat('[')
```

Comment:
Don't need to change now, but use string interpolation instead of chained concat.